boot/bootutil/src/bootutil_priv.h

```diff
@@ -82,8 +82,10 @@ struct boot_status {
     uint8_t swap_type;    /* The type of swap in effect */
     uint32_t swap_size;   /* Total size of swapped image */
 #ifdef MCUBOOT_ENC_IMAGES
+#define BOOT_UNINITIALIZED_KEY_FILL 0xFF
```

Comment:
The convenience for avoid duplicating the `ifdef` comes at the price of hurting the code organization which, for me, makes it not worth it.
Please, evaluate moving these definitions before the struct declaration.

Reply:
You mean `#define BOOT_UNINITIALIZED_KEY_FILL 0xFF` and `#define BOOT_UNINITIALIZED_TLV_FILL 0xFF`? Because ifdefs itself controls wether `uint8_t enckey[BOOT_NUM_SLOTS][BOOT_ENC_KEY_ALIGN_SIZE];` and `uint8_t enctlv[BOOT_NUM_SLOTS][BOOT_ENC_TLV_ALIGN_SIZE];` are included in struct declaration.

Reply:
> You mean `#define BOOT_UNINITIALIZED_KEY_FILL 0xFF` and `#define BOOT_UNINITIALIZED_TLV_FILL 0xFF`?

Yes, exactly those definitions.